client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -431,41 +446,42 @@ function mediaButton( editor ) {
 		editor.nodeChanged();
 	}
 
-	function toggleSizingControls( event ) {
+	function toggleSizingControls( increase, event ) {
 		if ( ! event.element || 'IMG' !== event.element.nodeName ) {
 			return;
 		}
 
+		const site = sites.getSelectedSite();
```

Comment:
This perpetuates https://github.com/Automattic/wp-calypso/projects/3

We [reference the Redux store in `renderModal`](https://github.com/Automattic/wp-calypso/blob/9853eb5f87cfe331140943385ba7b989a2833429/client/components/tinymce/plugins/media/plugin.jsx#L75) above. Maybe we should bump this to a top-level variable ([example](https://github.com/Automattic/wp-calypso/pull/8700/files)) so we can access it here to call on the [`getSelectedSite`](https://github.com/Automattic/wp-calypso/blob/9853eb5f87cfe331140943385ba7b989a2833429/client/state/ui/selectors.js#L13-L26) selector instead. Note that the object returned by `getSelectedSite` is not identical to that from `lib/sites-list`, but should only be lacking [functions on the Site object prototype](https://github.com/Automattic/wp-calypso/blob/master/client/lib/site/index.js). All other [computed attributes](https://github.com/Automattic/wp-calypso/blob/master/client/lib/site/computed-attributes.js) will be present. If switched to using Redux site object, verify that all paths using that object don't need its prototype function.